### PR TITLE
Updating readme to reflect large grammer updates

### PR DESCRIPTION
**Pull request by Rumyra**
We're starting to see an increasing number of prs with large grammer updates which change the meaning of the content and are difficult to review. This adds a note in the readme to address this for the future

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -359,16 +359,21 @@ for each logical set of changes that belong together.
...
-     changed and why, and the change is obviously a good thing), you can do
-     this in your pull request's description.
-   - If the pull request is complex (the changes and the reasoning behind
-     them need a bit more explanation), then the requestor should file an
-     issue describing the intended change first, and seek discussion/approval
-     as needed. When the time is right to submit the PR, they should
-     reference the issue (or an existing issue that describes the motivation
-     for the change) in the PR. You can reference an existing issue
-     using `#` followed by the issue's ID, for example `#1234`.
+  - If the pull request is simple (it is really clear what has been
```
Comment (fiji-flo) [inline README.md:362]: I think there's a space missing
```md
1. foo
  - bar
```
1. foo
  - bar

vs

```md
1. foo
   - bar
```
1. foo
   - bar

and prettier aligns the starting of new line with the fist line of list item

```md
- foo
  bar
```

**Comment by Ryuno-Ki** [comment]
Sounds good to me.

I'd love to hear @hamishwillee's opinion on this, too.

File: README.md [hunk 2]
```diff
@@ -360,15 +360,20 @@ for each logical set of changes that belong together.
...
-     reference the issue (or an existing issue that describes the motivation
-     for the change) in the PR. You can reference an existing issue
-     using `#` followed by the issue's ID, for example `#1234`.
+   them need a bit more explanation), then the requestor should file an
+   issue describing the intended change first, and seek discussion/approval
+   as needed. When the time is right to submit the PR, they should
+   reference the issue (or an existing issue that describes the motivation
+   for the change) in the PR. You can reference an existing issue
+   using `#` followed by the issue's ID, for example `#1234`.
+   - Pull requests should not contain large amounts of grammar updates.
```
Comment (hamishwillee) [inline README.md:372]: This should be its own numbered point (it is separate to the point on describing the scope of changes)
You can then bullet out the main points, which currently read as overly long.

Maybe something like:

1.  Avoid pull requests that deliver many (or only) grammatical "tweaks":
    - Due to resource constraints, these may be closed following cursory review!
    - Particular care is required if you don't have domain expertise. Seemingly insignificant alterations can alter the meaning of technical content. 
    - Grammatical changes that remove ambiguity or enhance accessibility of the technical content are welcomed!

Comment (hamishwillee) [reply README.md:372]: ^^^ please excuse excessive ! use ! of !

Comment (Rumyra) [reply README.md:372]: Thanks @hamishwillee - as it's already a nested list I'm going to leave it - however I _am_ going to remember this comment when I get around to tidying up all these notes (there's a plan to separate them out) 👍

File: README.md [hunk 3]
```diff
@@ -360,15 +360,20 @@ for each logical set of changes that belong together.
 improvement, or formatting/structural change), please describe why you're
 making the change and anything else we need to know about it.
    - If the pull request is simple (it is really clear what has been
-     changed and why, and the change is obviously a good thing), you can do
-     this in your pull request's description.
+   changed and why, and the change is obviously a good thing), you can do
```
[the remark this answers is not in the document]
Comment (Rumyra) [reply README.md:363]: It's a markdownlint thing - the tests won't pass if I don't have it like this 🤷🏻‍♀️

**Pull request merged**